src/test/java/com/exasol/cloudwatch/ExasolStatisticsTableMetricReaderTest.java

```diff
@@ -0,0 +1,147 @@
+package com.exasol.cloudwatch;
+
+import static com.exasol.cloudwatch.ExasolStatisticsTableMetric.QUERIES;
+import static com.exasol.cloudwatch.ExasolStatisticsTableMetric.USERS;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.time.*;
+import java.time.temporal.ChronoUnit;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.itsallcode.io.Capturable;
+import org.itsallcode.junit.sysextensions.SystemOutGuard;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import com.exasol.containers.ExasolContainer;
+
+@Testcontainers
+@ExtendWith(SystemOutGuard.class)
+class ExasolStatisticsTableMetricReaderTest {
+    @Container
+    private static final ExasolContainer<? extends ExasolContainer<?>> EXASOL = new ExasolContainer<>("7.0.5")
```

Comment:
You have two versions of docker container, here 7.0.5 and 7.0.4 in cloudwatch IT.